virttest/shared/cfg/guest-os/Windows.cfg

```diff
@@ -125,6 +125,8 @@
         rtc_drift = "slew"
         time_command = "echo TIME: %date% %time%"
         time_filter_re = "(?<=TIME: \w\w\w ).{19}(?=\.\d\d)"
+        Win11:
+            time_filter_re = "(?<=TIME: ).{19}(?=\.\d\d)"
```

Comment:
@YanhuiMa  If i understand correctly, this parameter is used by:
https://github.com/avocado-framework/avocado-vt/blob/3de21fc1f3b725609695545ee54f4fc514ce2b77/virttest/utils_test/__init__.py#L317

The usage is: `re.findall(time_filter_re, content)[0]`.

So, suggest use one pattern for all: `time_filter_re = 'TIME:\s+(?:\w{3}\s|)(\d{2}/\d{2}/\d{4}\s+\d{1,2}:\d{2}:\d{2})(?=\.\d{2})'`

Reply:
Yes, it works. And time_format also needs to been modified. I have added it for Win11